cucumber/policy/features/support/client.rb

```diff
@@ -0,0 +1,147 @@
+
+module ClientHelpers
+  attr_reader :result
+
+  class Client
+    @@admin_password = 'SEcret12!!!!'
+    @@account = ENV['CONJUR_ACCOUNT'] || 'cucumber'
+    @@appliance_url =  ENV['CONJUR_APPLIANCE_URL'] || 'http://conjur'
```

Comment:
Another of one of those "don't copy from our current codebase" landmines I mentioned during our last call, but basically consider `@@` to be a "never use" ruby feature.  I'll give you some more context on that in person, but there are always better alternatives we can use.

Reply:
Also, we should take the ENV var info as arguments to the constructor.